pyqtgraph/flowchart/Flowchart.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from ..Qt import QtCore, QtGui, QT_LIB
+from ..Qt import QtCore, QtGui, QT_LIB, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets, QtWidgets
```

Comment:
:laughing: When you want to be *absolutely* sure that your module is imported

Reply:
🤣 I did not, with my human eyes, look upon even a tenth of this PR. I expect that's an artifact of the iterative process by which I wrote the regex that did the work here, and if we rerun from scratch, it'll go away.